console/my-account/how-to/accept-invitation-new-scw-account.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to activate a new account after being invited to an Organization
+  description: This page explains how to activate a new account after being invited to an Organization
+  h1: How to activate a new account after being invited to an Organization
+  paragraph: This page explains how to change the color scheme of the Scaleway console
+tags: color-scheme change color theme dark-mode change-color
+dates:
+  validation: 2023-06-05
+  posted: 2023-06-15
+categories:
+  - console
+---
+
+When a Scaleway user invites you to join their Organization you receive an invitation via email. If do not yet have a Scaleway account upon receiving this email, you will not be able to effectively join the Organization.
```

Comment:
Missing the you: "If YOU do not have..."

Reply:
```suggestion
When a Scaleway user invites you to join their Organization you receive an invitation via email. If you do not yet have a Scaleway account upon receiving this email, you will not be able to effectively join the Organization.
```